pkg/sfu/mediaengine.go

```diff
@@ -0,0 +1,92 @@
+package sfu
```

Comment:
I think this is the code which was already living in `livekit-server` and hence probably not needed to be moved over.